### DISCUSSION TITLE
How to use livewire for   package development.

### DISCUSSION THREAD

**Discussion by magic-thomas** [livewire/livewire#2947] [1. Help]
``Hi. 

I use orchestra/canvas for package development.

And I want to use livewire for package view / component. 

So I included two packages in my package composer.json

`
"require-dev": {
        "orchestra/canvas": "^6.2"
    },
    "require": {
        "livewire/livewire": "^2.4"
    }
`

In my package root directory, I would like to run ' livewire:make <component-name>' 

How can I run this and make livewire component  ?

Or where should I set up livewire component / view in my package directory?

**Comment by andreapozza** [comment]
I think this page is missing in 3.x 
https://laravel-livewire.com/docs/2.x/package-dev
